<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btnWeek">
     <property name="text">
      <string>שבוע</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMonth">
     <property name="text">
      <string>חודש</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnThreeMonths">
     <property name="text">
      <string>3 חודשים</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
